### ISSUE TITLE
Skipping formatter plugin Phoenix.LiveView.HTMLFormatter because module cannot be found

### ISSUE THREAD

**Issue by sergiotapia** [phoenixframework/phoenix_live_view#1910]
### Environment

* Elixir version (elixir -v): Elixir 1.13.3 (compiled with Erlang/OTP 22)
* Phoenix version (mix deps): phoenix 1.6.6
* Phoenix LiveView version (mix deps): phoenix_live_view 0.17.7
* Operating system: Linux Mint
* Browsers you attempted to reproduce this bug on (the more the merrier): N/A
* Does the problem persist after removing "assets/node_modules" and trying again? Yes/no: N/A

### Actual behavior

Trying to use the new Heex formatter for my templates. This is a brand new project. Here are my deps:

```
{:phoenix, "~> 1.6.6"},
{:phoenix_live_view, "~> 0.17.7", override: true},

# .formatter.exs
[
  plugins: [Phoenix.LiveView.HTMLFormatter],
  import_deps: [:ecto, :phoenix],
  inputs: ["*.{heex,ex,exs}", "priv/*/seeds.exs", "{config,lib,test,seeds}/**/*.{heex,ex,exs}"],
  subdirectories: ["priv/*/migrations"]
]
```

When I run `mix format` I get this error, and my jumbled .html.heex file isn't formatted.

`Skipping formatter plugin Phoenix.LiveView.HTMLFormatter because module cannot be found`

I also tried to `mix compile` before running `mix format` and have the same error.

### Expected behavior

My heex template should be formatted properly.

**Comment by sergiotapia** [comment]
To fix this now, I had to use Liveview `master` branch from git in my `mix.exs` file.

```
--- {:phoenix_live_view, "~> 0.17.7"},
+++ {:phoenix_live_view, git: "https://github.com/phoenixframework/phoenix_live_view", override: true},
```

Then, I had to run `mix compile`, finally `mix format` worked. Without the `mix compile` step I would get the same error of "module not found".

Hope this helps someone. I will close this ticket.

**Comment by Sed9yDataBank** [comment]
> @sergiotapia i do not think this is needed anymore!
> 
> i was using this, and i faced some issues today:
> 
> `Could not compile :phoenix_live_view, no "mix.exs", "rebar.config" or "Makefile"` `could not compile dependency :phoenix_live_dashboard`
> 
> i removed it, then ran `mix format` and it is still working
> 
> `mix.exs` : `{:phoenix_live_view, "~> 0.17.5"}` `mix.lock` : ` "phoenix_live_view": {:hex, :phoenix_live_view, "0.17.9", ...}`

yes, issue is fixed in `0.17.8` https://github.com/phoenixframework/phoenix_live_view/commit/d7ca847f4bd2cce7b91b1c99e4b024d899a3ace5
